workflow/exp/exp.jet_conus3km

```diff
@@ -62,3 +62,7 @@ export NODES_MPASSIT="<nodes>4:ppn=20</nodes>"
 export NODES_UPP="<nodes>4:ppn=20</nodes>"
 #
 export WALLTIME_FCST="4:00:00"
+
+#
+export KEEPDATA=yes
```

Comment:
We want to separate config settings between setup-time and runtime. I had thought using exp.setup for setup-time config after our discussion and config files for runtime config. But now it looks we will get exp.setup files bigger and bigger and mix the setup-time config and runtime config together in exp.setup files.